src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java

```diff
@@ -104,7 +101,7 @@ public String getProperty(String propertyName) throws CheckstyleException {
     @Override
     public Configuration[] getChildren() {
         return children.toArray(
-                EMPTY_CONFIGURATION_ARRAY);
+                new Configuration[0]);
```

Comment:
This means you found defect in checkerframefork tool.
Please create issue on it, close this PR, make new PR with comments over xml suppression blocks with comment `<-- until #xxxxxx -->`